github_connector/models/github_team_partner.py

```diff
@@ -34,3 +34,22 @@ class GithubTeamPartner(models.Model):
     role = fields.Selection(
         selection=_ROLE_SELECTION, string="Role", required=True, readonly=True
     )
+
+    context_search_default_team_id = fields.Integer(
+        compute="_compute_context_search_default"
+    )
+    context_search_default_partner_id = fields.Integer(
+        compute="_compute_context_search_default"
+    )
+
+    def _compute_context_search_default(self):
```

Comment:
Don't need to decorate these (context dependent) methods with @api.depends_context? In this case @api.depends_context('search_default_team_id', 'search_default_partner_id')
ping @joao-p-marques @pedrobaeza

Reply:
Indeed.